serving/go.mod

```diff
@@ -0,0 +1,59 @@
+module github.com/llm-d/llm-d-inference-serving
+
+go 1.24.2
```

Comment:
There are security updates in later patch releases.
There is a new minor version.